python/pyarrow/_parquet.pxd

```diff
@@ -557,6 +557,7 @@ cdef extern from "parquet/arrow/writer.h" namespace "parquet::arrow" nogil:
 
         CStatus WriteTable(const CTable& table, int64_t chunk_size)
         CStatus NewRowGroup(int64_t chunk_size)
```

Comment:
As discussed this should be removed too. I think CI should just pass as this internal API is not being used.
```suggestion
```